pkg/controller/provider/validation.go

```diff
@@ -362,10 +362,10 @@ func (r *Reconciler) inventoryCreated(provider *api.Provider) error {
 
 func (r *Reconciler) handleServerCreationFailure(provider *api.Provider, err error) {
 	provider.Status.Phase = ConnectionAuthFailed
-	msg := fmt.Sprint("The OVA provider server creation failed -", err)
+	msg := fmt.Sprint("The OVA provider server creation failed: ", err)
 	provider.Status.SetCondition(
 		libcnd.Condition{
-			Type:     ConnectionAuthFailed,
+			Type:     ConnectionFailed,
```

Comment:
I've changed the type, it should be `ConnectionFailed` and not `ConnectionAuthFailed`
and included the changes you added here as well so they are not override.

Reply:
i don't get this comment.
do we have auth problems? only connection problems?
if we have only `ConnectionFailed` do we still need `ConnectionAuthFailed` few lines above?

which changes were added to not be overridden?

Reply:
> I've changed the type, it should be `ConnectionFailed` and not `ConnectionAuthFailed` 

right, ack

> and included the changes you added here as well so they are not override.

cool, but you dropped changes in the createPvForNfs and createPvcForNfs functions, intentionally?

Reply:
@ahadas no I didn't drop them, because the return now is same as it was in the function originally it doesn't catch it as a change anymore so its not displayed.

@liranr23 you right the above statues need to be also changed.